Improve GitHub bug report issue template (#13976)
fastlane/fastlane: .github/ISSUE_TEMPLATE/BUG_REPORT.md

### Issue Description
<!-- Please include what's happening, expected behavior, and any relevant code samples -->

##### Command executed

> <!-- The command you executed on the command line, that resulted in an error -->

Reviewer: You could remove the comma.

Author: ```suggestion
<!-- The command you executed on the command line that resulted in an error -->
```

##### Complete output when running fastlane, including the stack trace and command used
<!-- 

> You can use `--capture_output` as the last commandline argument for many commands to get that collected for you. Otherwise, please do it manually.

Reviewer: Space inside `commandline`.

Author: ```suggestion
You can use `--capture_output` as the last command line argument for many commands to get that collected for you. Otherwise, please do it manually.
```

Caution: The output of `--capture_output` could contain sensitive data such as application ids, certificate ids, or email addresses. Please make sure you double check the output and replace anything sensitive you don't wish to submit in the issue 
-->

<details>
  <pre> [REPLACE THIS WITH YOUR INFORMATION] </pre>
</details>

### Environment

<!-- 
Please run `fastlane env` and copy the output below. This will help us help you.

> If you used `--capture_output` option, please remove this block as it is already included there. 

Reviewer: `used the`?

Author: ```suggestion
If you used the `--capture_output` option, please remove this block as it is already included there. 
```
